sim/hunter/glyphs.go

```diff
@@ -0,0 +1,23 @@
+package hunter
+
+import (
+	"github.com/wowsims/cata/sim/core"
+	"github.com/wowsims/cata/sim/core/proto"
+)
+
+func (hunter *Hunter) ApplyGlyphs() {
+	if hunter.HasPrimeGlyph(proto.HunterPrimeGlyph_GlyphOfArcaneShot) {
+		hunter.AddStaticMod(core.SpellModConfig{
+			Kind:       core.SpellMod_DamageDone_Pct,
```

Comment:
https://wago.tools/db2/SpellEffect?build=4.4.0.54217&filter[SpellID]=%0956841&page=1 as Rain pointed out, this should be DamageDone_Flat

Reply:
Ah, I definitely thought that `SpellMod_DamageDone_Pct` was the additive mod, will fix it.